src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURE_ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FIGURE_ARRAY_SIZE];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < FIGURE_ARRAY_SIZE / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = FIGURE_ARRAY_SIZE / 2; i < FIGURE_ARRAY_SIZE; i++) {
```

Comment:
Use of a loop to create several random figures is good. Also, it's great that FigureSupplier is not created inside the loop.